yt_dlp/extractor/zee5.py

```diff
@@ -97,14 +97,14 @@ def _real_extract(self, url):
             (lambda x: x['hls'][0], lambda x: x['video_details']['hls_url']),
             str)
         formats = self._extract_m3u8_formats(
-            'https://zee5vodnd.akamaized.net' + m3u8_url.replace('/drm1/', '/hls1/') + token_request['video_token'],
+            'https://zee5vodnd.akamaized.net' + m3u8_url.replace('drm', 'hls') + token_request['video_token'],
```

Comment:
```suggestion
            'https://zee5vodnd.akamaized.net' + m3u8_url.replace('/drm', '/hls', 1) + token_request['video_token'],
```

to minimize the chance of any false replaces.